packages/orbit-components/config/build/consts.mts

```diff
@@ -14,7 +14,8 @@ export const COMPILE_IGNORE_PATTERNS = [
   // TODO: remove that after TW migration, tailwind is temporary folder
   "**/tmp-tailwind/**/*.{ts?(x),d.ts}",
   "**/*.d.ts",
-  "**/*.stories.*",
+  "**/*.ct.*",
+  "**/*.ct-story.*",
```

Comment:
🙏🏻 add also to ignore for type declarations [here](https://github.com/kiwicom/orbit/blob/master/packages/orbit-components/config/build/consts.mts#L24)

Reply:
fixed